MSGraphSDKNXOAuth2/Auth/NXOAuth2AuthenticationProvider.m

```diff
@@ -188,9 +188,17 @@ - (void)loginWithViewController:(UIViewController*)viewController
     }];
 }
 
-//- (BOOL)loginSilent {
-//    // Placeholder: Load an existing account from the account store
-//}
+- (BOOL)loginSilent {
+    // Placeholder: Load an existing account from the account store
+    int accountCount = [[NXOAuth2AccountStore sharedStore] accounts].count;
+    
+    if (accountCount == 0) {
+        return NO;
+    } else {
+        self.userAccount = [[NXOAuth2AccountStore sharedStore] accounts][accountCount - 1];
```

Comment:
Is NXOauth2 documented to rearrange its account list in MRU fashion? If not it might be prudent to track that in shared preferences or something.